libs/tailwindcss-config/src/theme.js

```diff
@@ -10,6 +10,33 @@ module.exports = {
   colors: {
     transparent: 'transparent',
     current: 'currentColor',
+    bullish: '#26FF8A',
+    bearish: '#ED1515',
+    vega: {
+      yellow: '#EDFF22',
+      pink: '#FF2D5E',
+      green: '#00F780',
+    },
+    red: {
+      DEFAULT: '#ED1515',
+      transparent: 'rgba(255, 38, 65, 0.3)',
+      vega: '#FF261A',
+    },
+    green: {
+      DEFAULT: '#26FF8A',
+      transparent: 'rgba(38, 255, 138, 0.3)',
+      dark: '#246340', // Same as GREEN_TRANSPARENT given a background of #1f1f1f
+      vega: '#00F780',
+    },
+    text: '#C7C7C7',
+    deemphasise: '#8A9BA8',
+    gray: {
```

Comment:
chore (blocking): Lets try to avoid adding all these grays. If they've been added due to new colors coming in via apps/token lets update to just use our current grayscale. Eg `text-white-40`. Use the closest one you can.

Reply:
we need to find grays at some point. either we choose white-40 or black-40, we constantly need to add extra logic for this color to switch to be visible on dark/light themes.